connectors/snowflake/manifest.json

```diff
@@ -0,0 +1,170 @@
+{
+  "name": "SQL Importer",
```

Comment:
Do we want to call it `Snowflake Importer`?

Reply:
done